header.html

```diff
@@ -146,6 +146,18 @@ <h2>About Myself</h2>
                 </ul>
                 </div>
             </section>
+            <!-- Contact form -->
+            <section class="form-group">
+                <img src="img/contact-form-shapes-mobile.svg" alt="Background shapes design" class="form-bg-img">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
+                <form class="form-input">
+                    <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" required>
+                    <input type="email" name="Email" id="email" placeholder="Enter your email" required>
+                    <textarea name="comments" id="comments" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                    <button class="btn" type="submit">Get It Touch</button>
+                </form>
```

Comment:
- [ ] Kindly add the [form service](https://formspree.io/html/) to collect data from forms, please check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) to see how to integrate this feature,